package.json

```diff
@@ -6,8 +6,8 @@
   "scripts": {
     "test": "nyc mocha --exit --require @babel/register --timeout 60000 ./server/test/**/*.spec.js",
     "coverage": "nyc report --reporter=text-lcov | coveralls",
-    "start": "npm run build && node build/index.js",
-    "build": "babel index.js --out-dir build",
+    "start": "node build/index.js",
+    "build": "babel index.js -d build && babel server -d build/server",
```

Comment:
This looks good. However, recursively remove the build folder before transpiling so that it generates the build folder afresh.

Reply:
Alright....will implement that now

Reply:
Implemented this